experimental/tf_proxy/server/src/main.rs

```diff
@@ -74,9 +76,28 @@ async fn main() -> anyhow::Result<()> {
     let address = SocketAddr::from((Ipv6Addr::UNSPECIFIED, opt.port));
     wait_for_socket(SOCKET).await?;
     let backend = grpc::BackendConnection::connect(SOCKET).await;
-    let handler = AttestationServer::create(Vec::new(), backend)?;
+    let client = backend.create_client();
+
+    // Create fake configuration info for now, as it cannot be empty for the attestation
+    // handshake.
+    // TODO(#2420): Remove once Java client can work without the configuration info.
```

Comment:
What would we send instead of the configuration info? Some things are still necessary even from this other binary, e.g. some signature, inclusion proof, etc. cc @rbehjati

Reply:
In future, the server will as well send a LogEntry (e.g., from Rekor) to the client. I think it is easier to have it as part of `ConfigurationInfo`, but it can also be a separate object. 

I am not sure if we want to skip sending configuration info. Perhaps, the Java client is fine but the Rust client must be updated to fail when no Configuration info is provided.